internal/pkg/services/load-balancer/utils/utils_test.go

```diff
@@ -0,0 +1,121 @@
+package utils
+
+import (
+	"testing"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/stackitcloud/stackit-sdk-go/services/loadbalancer"
+)
+
+func fixtureGetDefaultPayload(mods ...func(payload *loadbalancer.CreateLoadBalancerPayload)) *loadbalancer.CreateLoadBalancerPayload {
+	payload := &loadbalancer.CreateLoadBalancerPayload{
+		ExternalAddress: utils.Ptr(""),
+
+		Listeners: &[]loadbalancer.Listener{
+			{
+				DisplayName: utils.Ptr(""),
+				Name:        utils.Ptr(""),
+				Port:        utils.Ptr(int64(0)),
+				Protocol:    utils.Ptr(""),
+				ServerNameIndicators: &[]loadbalancer.ServerNameIndicator{
+					{
+						Name: utils.Ptr(""),
+					},
+				},
+				TargetPool: utils.Ptr(""),
+				Tcp: &loadbalancer.OptionsTCP{
+					IdleTimeout: utils.Ptr(""),
+				},
+				Udp: &loadbalancer.OptionsUDP{
+					IdleTimeout: utils.Ptr(""),
+				},
+			},
+		},
+		Name: utils.Ptr(""),
+		Networks: &[]loadbalancer.Network{
+			{
+				NetworkId: utils.Ptr(""),
+				Role:      utils.Ptr(""),
+			},
+		},
+		Options: &loadbalancer.LoadBalancerOptions{
+			AccessControl: &loadbalancer.LoadbalancerOptionAccessControl{
+				AllowedSourceRanges: &[]string{
+					"",
+				},
+			},
+			EphemeralAddress: utils.Ptr(false),
+			Observability: &loadbalancer.LoadbalancerOptionObservability{
+				Logs: &loadbalancer.LoadbalancerOptionLogs{
+					CredentialsRef: utils.Ptr(""),
+					PushUrl:        utils.Ptr(""),
+				},
+				Metrics: &loadbalancer.LoadbalancerOptionMetrics{
+					CredentialsRef: utils.Ptr(""),
+					PushUrl:        utils.Ptr(""),
+				},
+			},
+			PrivateNetworkOnly: utils.Ptr(false),
+		},
+		PrivateAddress: utils.Ptr(""),
+		TargetPools: &[]loadbalancer.TargetPool{
+			{
+				ActiveHealthCheck: &loadbalancer.ActiveHealthCheck{
+					HealthyThreshold:   utils.Ptr(int64(0)),
+					Interval:           utils.Ptr(""),
+					IntervalJitter:     utils.Ptr(""),
+					Timeout:            utils.Ptr(""),
+					UnhealthyThreshold: utils.Ptr(int64(0)),
+				},
+				Name: utils.Ptr(""),
+				SessionPersistence: &loadbalancer.SessionPersistence{
+					UseSourceIpAddress: utils.Ptr(false),
+				},
+				TargetPort: utils.Ptr(int64(0)),
+				Targets: &[]loadbalancer.Target{
+					{
+						DisplayName: utils.Ptr(""),
+						Ip:          utils.Ptr(""),
+					},
+				},
+			},
+		},
+	}
+	for _, mod := range mods {
+		mod(payload)
+	}
+	return payload
+}
+
+func TestGetDefaultPayload(t *testing.T) {
```

Comment:
this unit test is basically not testing anything and it's just adding maintenance effort if some value is updated on the default payload. It makes more sense to make the default payload a const of the generate-payload command and remove this `utils` pkg entirely